### PR TITLE
Feature/image

### PR DESCRIPTION
**Pull request by tarekcham**

### CODE REVIEW

**Comment by orwa337** [comment]
there is a problem 
the acceptsingledelivery page is totally broken!!!!

**Comment by NachoMerino** [comment]
@tarekcham any progresswith this?

**Comment by tarekcham** [comment]
the fix the error plz review the code

File: frontend/src/components/master-components/UserDetails.js [hunk 1]
```diff
@@ -69,14 +72,14 @@ export default class UserDetails extends Component {
   };
 
   editpicHandler = () => {
+    console.log('editpicHandler')
```
Comment (NachoMerino) [inline frontend/src/components/master-components/UserDetails.js]: DELETE IT

File: frontend/src/components/master-components/AcceptSingleDelivery.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ export default class AcceptSingleDelivery extends Component {
     }
     return (
       <div className="accept-single-delivery main">
+      {console.log(this.props.orderer.firstname)}
```
Comment (NachoMerino) [inline frontend/src/components/master-components/AcceptSingleDelivery.js]: DELETE IT

Comment (tarekcham) [reply frontend/src/components/master-components/AcceptSingleDelivery.js]: i deleted them :)

**Comment by NachoMerino** [comment]
Hi, the picture is not a circle, it is an egg form, can you please change it  into a circle?
egg-form

**Comment by tarekcham** [comment]
now they are matching :)

**Comment by tarekcham** [comment]
review it agian

**Comment by NachoMerino** [comment]
eggform2
Still an Egg, please, can somebody review too and check if still like that
@orwa337 
@Versioncrowd 
@laithmassoud

**Pull request closed without merging**